plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -922,6 +918,12 @@ public Optional<io.trino.spi.type.Type> getSupportedType(ConnectorSession sessio
         if (type instanceof TimeType) {
             return Optional.of(TIME_MICROS);
         }
+        if (type instanceof TinyintType || type instanceof SmallintType){
+            return Optional.of(INTEGER);
+        }
+        if (type instanceof CharType) {
+            return Optional.of(CharType.createCharType(((CharType) type).getLength()));
+        }
```

Comment:
This is same as The `CharType` right ?

Reply:
We could translate `char` to `varchar` right ? Does `Iceberg` has any types specific to `Char` ?